web/src/elements/ak-locale-context/ak-locale-context.ts

```diff
@@ -41,6 +42,9 @@ export class LocaleContext extends LitElement {
 
     setLocale: LocaleSetter;
 
+    @state()
+    userLocale = "";
```

Comment:
We don't have a good "subscribe to user changes," so I'm having to keep this one local.